src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -1,8 +1,81 @@
 // React Imports
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 // MUI Imports
-import Typography from '@mui/material/Typography';
+import { TextField, Button } from '@mui/material';
+import { useCivicProfile } from '@hooks';
 
-const HousingInfo = () => <Typography>Housing Info</Typography>;
+const HousingInfo = () => {
```

Comment:
Will these forms require JSDocs?

Reply:
Think it'll be good to include them, perhaps a small snippet of what fields users likely be filling out in these forms

Reply:
Added